src/main/resources/embedding-providers-config.yaml

```diff
@@ -104,6 +104,48 @@ stargate:
                   help: "Vector dimension to use in the database and when calling Azure OpenAI."
             - name: text-embedding-ada-002
               vector-dimension: 1536
+        bedrock:
+          # see https://learn.microsoft.com/en-us/azure/ai-services/openai/reference
```

Comment:
Probably not right URL?

Reply:
Yes, URL is not an optional field. Both empty string and null not accepted. Need a separate PR to handle this since there is no URL for Amazon Bedrock.